Articles/Packages/Extending/index.md

```diff
@@ -0,0 +1,80 @@
+# Packages
+
+## What is a Package?
+
+A package extends Umbraco to provide additional functionality to editors, developers, site visitors, and all other types of users of Umbraco. It can impact one or more of these groups of people depending on the type of package.
+
+An Umbraco Package can be many things, but are generally characterized by:
```

Comment:
```suggestion
An Umbraco Package can be many things, but are generally characterised by:
```

Assuming UK English is used of US English?  If not, ignore this comment.

Reply:
I'm not really sure to be honest, got any pointers @sofietoft

Reply:
Actually, the general language we want to use going forward (not everything is updated yet 😅 ) is **US English**.

I'll make sure it's added to our style guide 💪